x-pack/plugin/rollup/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
   compileOnly project(path: xpackModule('ilm'))
   compileOnly project(path: xpackModule('mapper-aggregate-metric'))
   testImplementation(testArtifact(project(xpackModule('core'))))
+  testImplementation project(':modules:lang-painless')
```

Comment:
Just double checking, what part of the test requires the usage of painless?

Reply:
Sorry, this was a left-over...